about.html

```diff
@@ -0,0 +1,162 @@
+                </ul>
+            </div>
+        </div>
+        <div class="mobile-menu-list invisible">
+            <div class="navbar-mobile">
+                <div class="logo">
+                    <a href="index.html"><img class="logo-image" src="images/logo.svg"></a>
+                </div>
+                <div class="menu-icon">
+                    <a id="hide-menu" class="fa far fa-window-close" href="#"></a>
+                </div>
+               
+            </div>
+            <ul class="menu-list">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Documentation</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+    
+        </div>
+
+        <div class="navbar-desktop invisible"></div>
+    </nav>
+    <main class="main about-main">
+        <section class="headline">
+            <div class="headline-opacity">
+                <header class="headline-header">
+                    <h1>Africa's Talking Open Hackathon: FinTech</h1>
+                </header>
+                <div class="headline-desc">
+                    <p>
+                        Join us at our open hackathon every last Thursday of the month.
+                         Come build with us, present feature request, give feedback or 
+                         just contribute to our open source API's and SDKs. 
+                         Africa's Talking community allows developers to learn skills for the modern-day African Developer. 
+                         We are language and framework agnostic. All developers are welcome    
+                    </p>
+                </div>
+                <div class="when-where">
+                    <div class="when">
+                        <div class="when-header">
+                            <a class="fa far fa-calendar" href="#"></a>
+                            <h3> When</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Thursday, Jun 30 2022<br>
+                                9:00 AM - 5:30 PM (EAT)
+                            </p>
+                        </div>
+                    </div>
+                    <div class="when">
+                        <div class="when-header">
+                            <a class="fa fa-map-marker" href="#"></a>
+                            <h3> Where</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Africa's Talking HQ<br>
+                                Mzima Springs Rd, Lavington, Nairobi<br>
+                                Nairobi, 00100<br>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] The top section (this section) should have a centered column that includes a page title, a text block with the page description, and a line with contact information or any other prompt under it, kindly align all the elements in this section in the center and consider changing the background to one with a white accent

|Your design|CC design|
|-|-|
|![Screenshot_20220623_200721](https://user-images.githubusercontent.com/25227022/175377962-7134f8c3-1759-4f36-8935-10cf34247a29.png)|![Screenshot_20220623_200645](https://user-images.githubusercontent.com/25227022/175377896-d73ec275-8b14-4074-854f-7502323f8aeb.png)|